lib/html_tag.js

```diff
@@ -6,7 +6,7 @@ function htmlTag(tag, attrs, text) {
   var result = '<' + tag;
 
   for (var i in attrs) {
-    if (attrs[i]) result += ' ' + i + '="' + attrs[i] + '"';
+    if (attrs[i] || attrs[i] === '') result += ' ' + i + '="' + attrs[i] + '"';
```

Comment:
I suggest we use stricter `attrs[i] !== undefined && attrs[i] !== null` on this check.

Reply:
There is an error already if an attribute is not defined.  Should I add the test?
```
it('tag + empty attr', function() {
    try {
      htmlTag('img', {
        src: 'http://placekitten.com/200/300',
        hidden
      });
    } catch (err) {
      err.should.have.property('message', 'hidden is not defined');
    }
  });
```
or
```
it('tag + empty attr', function() {
    try {
      htmlTag('img', {
        src: 'http://placekitten.com/200/300',
        alt: undefined
      });
    } catch (err) {
      err.should.have.property('message', 'alt is not defined');
    }
  });
```